locale/en-US/translation.js

```diff
@@ -117,6 +117,7 @@ const translation = {
     settingsToggleTestnet:"",       //Toggle Testnet Mode
 
     //Transparency Report
+    transparencyReport: "<center>--- Transparency Report ---</center><br>",
```

Comment:
We should keep any HTML out of here to simplify it for translators (would simply concatenate the HTML elements around the translation string).